roles/confluent.test/molecule/rbac-centralized-mds-kerberos-debian/molecule.yml

```diff
@@ -0,0 +1,202 @@
+### rbac-centralized-mds-kerberos-debian
+##
+##  This Scenario requires that rbac-mds-kerberos-debian scenario has been run successfully prior to running it.
```

Comment:
Might want to say the converge phase has completed, run successfully to me might mean test completed